tools/build_api.py

```diff
@@ -52,6 +52,11 @@
 
 RELEASE_VERSIONS = ['2', '5']
 
+SMALL_C_LIB_CHANGE_WARNING = (
```

Comment:
Separating the message format from its arguments doesn't really make sense here. Since this is used only once, at the print, it makes more sense to put the text template there.